go.mod

```diff
@@ -4,9 +4,8 @@ go 1.13
 
 require (
 	emperror.dev/errors v0.4.2
-	github.com/Azure/go-autorest/autorest/azure/auth v0.4.0 // indirect
+	github.com/Azure/go-autorest/autorest/azure/auth v0.4.2 // indirect
```

Comment:
@baluchicken would it make sense to run `go get -u=patch -t` to update all deps to latest patch versions? I use to do this in my go projects so I stay up-to-date in a sane way.